tests/unittest/test_s3_crashstorage.py

```diff
@@ -130,6 +134,7 @@ def test_region_and_bucket_with_periods(
         client.rebuild_app(
             {
                 "CRASHSTORAGE_CLASS": "antenna.ext.s3.crashstorage.S3CrashStorage",
+                "CRASHSTORAGE_ENDPOINT_URL": "http://fakes3:4569",
```

Comment:
Missing this caused this test to fail. This was a mystery I bumped into a while back, but couldn't figure out. What's going on is that since `CRASHSTORAGE_ENDPOINT_URL` isn't defined by default, then this would pick up AWS urls. Then we changed it to use the environment for configuration, so then it was an empty string and now it's set to this.